src/status_im/contexts/wallet/collectible/view.cljs

```diff
@@ -70,6 +72,32 @@
       :accessibility-label :share-details
       :label               (i18n/label :t/share-details)}]]])
 
+(defn options-drawer
+  [images index]
+  (let [{:keys [image]} (nth images index)
+        uri             (url/replace-port image (rf/sub [:mediaserver/port]))]
+    [quo/action-drawer
+     [[{:icon                :i/link
+        :accessibility-label :view-on-etherscan
+        :label               (i18n/label :t/view-on-eth)}]
```

Comment:
we have some follow up issues made to add these actions? 🤔

Reply:
Not yet, will add